templates/endpoints_v2/README.ipynb

```diff
@@ -47,9 +48,41 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
```

Comment:
I think it would serve better ƒor the future if we don't allow this code to be runnable. Just make it bash formatted